models/models_methods_test.go

```diff
@@ -660,7 +660,7 @@ workflows:
 
 			warns, err := config.Validate()
 			if len(tt.wantWarns) > 0 {
-				require.Equal(t, tt.wantWarns, warns)
+				require.ElementsMatch(t, tt.wantWarns, warns)
```

Comment:
Flaky test? 🥲

Reply:
Yep. All the test cases used a single item array so the `Equal` was fine. I added a new test case with two items and the original PR was green. Merged it and found out that you should not use `Equal` to compare arrays. 😄